tests/test_function_manager.py

```diff
@@ -9,10 +9,9 @@
 import os
 test_location = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', 'binaries', 'tests')
 
+PROJECT = angr.Project(os.path.join(test_location, "x86_64", "fauxware"))
```

Comment:
We should not be creating objects like this in the global scope, these will end up getting wastefully evaluated many times. Nose supports pre and post functions though for the purpose of minimizing redundant code.

Reply:
I was not trying to minimize redundant code, I was trying to minimize redundant runs.
Did not see the point of reloading the project at each function...

Can you explain why this would get evaluated many times?